infra/modules/azure_cosmos_account/examples/complete/main.tf

```diff
@@ -25,7 +25,7 @@ module "cosmos_db" {
 
   secondary_geo_locations = [
     {
-      location          = "italynorth"
+      location          = "westeurope"
```

Comment:
why this change?

Reply:
because there was an error, the primary and secondary region cannot match